views-vues/assurance-level-requirement/resultTable.css

```diff
@@ -0,0 +1,39 @@
+.table-results {
+  width:1000px;
+}
+
+
+.table-results th, .table-results td, .table-results thead {
```

Comment:
Can these type of styles be achieved with the built in WET styles?

Reply:
Yes, turns out they can!
The only thing is that the table boarders don't stand out enough, so I just need to set their colour.

Should I use the style tag to implement that?
```
<style>
  .table-results th, .table-results td {
    border-color: #000 !important;
  }
</style>
```

Reply:
The PDF print out looks fine with the grey boarders.

Added some style attributes to certain parts of the table (head, body)